test/HealthChecks.OpenIdConnectServer.Tests/HealthChecks.OpenIdConnectServer.approved.txt

```diff
@@ -1,17 +1,17 @@
-namespace HealthChecks.IdSvr
+namespace HealthChecks.OpenIdConnectServer
```

Comment:
It's a breaking change, but since we want to release a next major version it's the best moment to do that.